### PR TITLE
Add the ability to specify the config file path via environment manager

### PR DESCRIPTION
**Pull request by scelfo**
This should make it possible to easily run hubibot inside of TrueNAS as an app via the "Launch Docker Image" flow which allows specifying environment variables and mount paths to datasets.

Note: I am not sure how to test this because when I run docker build I end up getting an error about permissions to /home/hubibot/.local due to the pip install command in the Dockerfile. I've never built a docker container so I'm likely running into some user error issues.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -473,16 +474,19 @@ def run(self) -> None:
         self.telegram.updater.start_polling()
         self.telegram.updater.idle()
 
-
-CONFIG_FILE = "config.yaml"
+env_config_file = os.getenv('HUBIBOT_CONFIG_FILE')
```
Comment (Nickolaim) [inline main.py:477]: what about this version of code? (I am not discussing semantic)

```python
env_config_file  = os.getenv("HUBIBOT_CONFIG_FILE", Path(__file__).with_name("config.yaml"))
```

**Comment by scelfo** [comment]
I had tried testing this change by building my own docker image but I ran into a permission issue. I've filed https://github.com/vdbg/hubibot/issues/51 with details about the problem.

File: main.py (hunk 1 above)
Comment (scelfo) [reply main.py:477]: Yeah that's clearly better, I had forgotten that os.getenv lets you specify a default. I'll update the pull request a bit later today and send it back your way.

**Comment by vdbg** [comment]
code change looks good. Could you update README.md to document the env var support and some of its potential uses?

**Comment by scelfo** [comment]
I put a quick update in the README.md. Once we get this into the latest docker image I'll verify that it works and take a screenshot of the configuration screen showing exactly what to set in TrueNAS.

**Comment by scelfo** [comment]
all set, let me know if you want me to add anything else

**Review by vdbg — APPROVED** [review]

**Comment by vdbg** [comment]
looks good thx for making these changes

**Pull request merged**